src/test/groovy/edu/ie3/datamodel/io/csv/BufferedCsvWriterTest.groovy

```diff
@@ -78,4 +79,33 @@ class BufferedCsvWriterTest extends Specification {
 		def e = thrown(SinkException)
 		e.message == "The provided data does not meet the pre-defined head line elements 'a,b,c'."
 	}
+
+	def "The buffered csv writer writes out content in the order specified by the headline elements"() {
+		given:
+		def targetFile = FilenameUtils.concat(tmpDirectory.toString(), "order_test.csv")
+		def writer = new BufferedCsvWriter(targetFile, ["c", "b", "a"] as String[], ",", false)
+		writer.writeFileHeader()
+		def content = [
+				"c": "z",
+				"a": "x",
+				"b": "y"
+		]
```

Comment:
Maybe slightly more expressive headline-value-pairs may improve readability of this? Quite hard to check the test  lines in 108 & 109